### PR TITLE
feat: support variate

### PR DESCRIPTION
**Pull request by Aganivi**
## What's Changed in this PR

*Describe the change in this PR*

## Checklist

- [x] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

- [x] Before merge, `**/variates.yml` should be update.

### CODE REVIEW

File: docs/.vitepress/config/setting.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "LATEST_VERSION": "v0.7",
   "websiteMap": { 
-    "en": "https://docs.greptime.com", 
+    "en": "https://feat-variate.greptime-docs.pages.dev",
```
Comment (tisonkun) [inline docs/.vitepress/config/setting.json:4]: Remember to revert before merge.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # GreptimeDB Documentation
 
 <p align="center">
-    <img src="./docs/public/logo-text-padding.png" alt="GreptimeDB Logo" width="400px">
+    <img src="./docs/public/logo-text-padding.png" alt="GreptimeDB Logo" width="400">
```
Comment (tisonkun) [inline README.md:4]: This seems unrelated changes?

File: docs/.vitepress/config/setting.json (hunk 1 above)
Comment (Aganivi) [reply docs/.vitepress/config/setting.json:4]: of course

**Review by nicecui — APPROVED** [review]

**Review by sunchanglong — APPROVED** [review]
